tests/app/Application/Services/OpenWalletServiceTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Tests\app\Application\Services;
+
+use App\Application\DataSources\UserDataSource;
+use App\Application\DataSources\WalletDataSource;
+use App\Application\Services\OpenWalletService;
+use App\Domain\User;
+use Mockery;
+use Tests\TestCase;
+
+class OpenWalletServiceTest extends TestCase
+{
+    private UserDataSource $userDataSource;
+    private WalletDataSource $walletDataSource;
+    private OpenWalletService $openWalletService;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->userDataSource = Mockery::mock(UserDataSource::class);
+        $this->walletDataSource = Mockery::mock(WalletDataSource::class);
+        $this->openWalletService = new OpenWalletService($this->userDataSource, $this->walletDataSource);
+    }
+
```

Comment:
In these test what is the SUT we are trying to test?
The use of the double of a test in this case Mockery is already a type of test.
Is necessary to test in this class if the response of the service is a null or `wallet_1`?